pom.xml

```diff
@@ -213,6 +213,30 @@
       <version>${plexus-io.version}</version>
       <scope>provided</scope>
     </dependency>
+    <!-- 
+     | Sling-Initial-Content
+     -->
+    <dependency>
+      <groupId>org.apache.sling</groupId>
+      <artifactId>org.apache.sling.jcr.contentloader</artifactId>
+      <version>2.4.1-SNAPSHOT</version>
```

Comment:
We'll need those dependencies at runtime - hence, they need to have scope provided so that they get picked up in the assembly.

Reply:
Done in https://github.com/apache/sling-org-apache-sling-feature-cpconverter/pull/74/commits/a639f016e7c5a0e6e589527a6c5328dea346e1ad.

Reply:
thanks